assets/src/hooks/useVehiclesForRunIds.ts

```diff
@@ -0,0 +1,30 @@
+import { Socket } from "phoenix"
+import { nullableParser } from "../api"
+import {
+  VehicleOrGhostData,
+  vehicleOrGhostFromData,
+} from "../models/vehicleData"
+import { VehicleOrGhost } from "../realtime"
+import { RunId } from "../realtime"
+import { useChannel } from "./useChannel"
+
+const parser = (data: VehicleOrGhostData[]): VehicleOrGhost[] =>
+  data.map(vehicleOrGhostFromData)
+
+const useVehiclesForRunIds = (
```

Comment:
this too could be folded into `SwingsView`

Reply:
or rather, into `swings_view.tsx`, in a helper function

Reply:
So the thing with `useVehiclesForRunIds` is that it's also used by the `useVehicleForNotification` hook, so even if it's small it's common to both places (I actually created this module by factoring out some of the code from `useVehicleForNotification`).

Reply:
ah OK I missed the other usage